lib/bindings/AMQPBinding.js

```diff
@@ -161,17 +163,24 @@ function start(callback) {
                     }
                 });
                 conn.on('close', function() {
-                    config.getLogger().error(context, 'reconnecting');
-                    if (numRetried <= retries) {
-                        numRetried++;
-                        return setTimeout(createConnection,
-                                          retryTime * 1000,
-                                          callback);
+                    // If amqpConn is null, the connection has been closed on purpose
```

Comment:
I understand that the changes applied to AMQP binding loging are the same as the ones discussed for the IOTA-JSON agent at https://github.com/telefonicaid/iotagent-json/pull/339/files#r230817939, isn't it?

Reply:
Yes, the changes are exactly the same

Reply:
NTC